whctools/signals.py

```diff
@@ -0,0 +1,2 @@
+# TODO @@@ Need a Signal check on member state change to Alumni to remove from all ACLS
```

Comment:
Yeah I would look into how the discord, Mumble etc hooks work. So you can do an ACL check on any member whos state changes to see if they also appear in the WHC database and then flag them.
We are planning on adding the Discord bot to AA, so you could also then send a Discord message to the appropriate channel.